public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>My Music Library</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="css/styles.css"/>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Mirza">
+</head>
+<body>
+  <div class="container">
+    <header>
+      <div class="logo">
+      <a href="/" class="logo__link">
+          <img src="img/purpsmiley.jpg" alt="My Music Library"/>
+      </a>
+      </div>
+      <h1 class="pageHeading">
+        <span class="pageHeading__word">My</span>
+        <span class="pageHeading__word">Music</span>
+        <span class="pageHeading__word">Library</span>
+      </h1>
+      <nav id="libraryNav">
+        <ul class="libraryNav__list">
+          <li class="libraryNav__item">
+            <a id="completeLib" href="/complete-library" class="navLink">Complete Library</a>
+          </li>
+          <li class="libraryNav__item">
+            <a id="allArt" href="/all-art" class="navLink">All Album Art</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section role="main">
+      <div id="results"></div>
+    </section>
+  </div>
+
```

Comment:
You index.html never loads any scripts, so if your server were able to run, there would never be any client-side application running.